src/store/flow/utils/importHelpers.ts

```diff
@@ -49,6 +49,64 @@ export function mergeFlowContainer(
   return existingFlowContainerStore
 }
 
+interface CreateFlowStackBlock {
+  uuid: string;
+  type: string;
+  config: {
+    flow_id: string;
+  };
+}
+
+interface CreateFlowStackFlow {
+  uuid: string;
+  blocks: CreateFlowStackBlock[];
+}
+
+interface CreateFlowStackData {
+  flows: CreateFlowStackFlow[];
+}
+
+export function createContainerFlowStack(json_data: any, flow_stack: string[]): void {
+  const flows = json_data.flows;
+  const visitedFlows: string[] = [];
+
+  function process_flow(flow: CreateFlowStackFlow): void {
+    visitedFlows.push(flow.uuid);
+    // eslint-disable-next-line no-restricted-syntax
+    for (const block of flow.blocks) {
+      if (block.type === 'Core.RunFlow') {
+        const flow_id: string = block.config.flow_id;
+        const child_flow: CreateFlowStackFlow | undefined = flows.find((f: { uuid: string }) => f.uuid === flow_id);
+        if (child_flow && !visitedFlows.includes(child_flow.uuid)) {
+          process_flow(child_flow);
+        }
+      }
+    }
+    if (!flow_stack.includes(flow.uuid)) {
+      flow_stack.push(flow.uuid);
+    }
+  }
+
+  // eslint-disable-next-line no-restricted-syntax
+  for (const flow of flows) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+    if (!visitedFlows.includes(flow.uuid)) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+      process_flow(flow);
+    }
+  }
+}
+
+export function getLastItemFromContainerFlowStack(flow_stack: string[]): string {
+  return flow_stack[flow_stack.length - 1];
+}
+export function createContainerFlowStackAndReturnLastItem(json_data: any): string {
+    const flow_stack: string[] = [];
+    if (isEmpty(json_data)) { return ''; }
+    createContainerFlowStack(json_data, flow_stack);
+    return getLastItemFromContainerFlowStack(flow_stack);
+}
```

Comment:
I would also recommend using a more functional-programming style, like:
```
export function createContainerFlowStackAndReturnLastItem(json_data: any): string {
    if (isEmpty(json_data)) {
        return ''
    }
    const flowStack = createContainerFlowStack(json_data)
    return flowStack.pop()!
}
```
Here the `createContainerFlowStack` is taking an input and returning a value, rather than mutating the input.
Also, as the `flowStack` is not needed for anything else, we can `pop` the last item - that's simpler.

Reply:
That would mean refactoring the `createContainerFlowStack`

Reply:
Thanks, Bulat, refactored createContainerFlowStack to return